elements/designer-frame/designer-frame.html

```diff
@@ -88,7 +79,7 @@
               new ClientConnection(window, this._frame.contentWindow);
         }.bind(this));
 
-        this._frame.src = getDocumentUrl(this.document);;
+        this._frame.src = getDocumentUrl(this.documentInfo.domDocument);
```

Comment:
Note: object URLs leak if you don't revoke them https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL